documentation/dotnet-trace-instructions.md

```diff
@@ -0,0 +1,118 @@
+# Trace for performance analysis utility (dotnet-trace)
+
+The dotnet-trace is a cross-platform CLI global tool that enables the collection of .NET Core traces of a running process without any native profiler involved. It is built around the EventPipe technology of the .NET Core runtime as an alternative to ETW on Windows and LTTNG on non-Windows.
+
+## Installing dotnet-trace
+
+The first step is to install the dotnet-trace CLI global tool. This requires the 2.1 .NET Core SDK to be installed.
+
+<!-- TODO: Make package globally available. -->
+```cmd
+$ dotnet tool install --global dotnet-trace
```

Comment:
You should use this command to install for now. When we push to nuget, the we will drop the --add-source from docs. dotnet-trace and dotnet-collect are already being pushed to the dotnet-core feed like dotnet-dump/dotnet-sos.

    $ dotnet tool install -g dotnet-dump --version 1.0.3-preview5.19217.3 --add-source https://dotnetfeed.blob.core.windows.net/dotnet-core/index.json
    You can invoke the tool using the following command: dotnet-dump
    Tool 'dotnet-dump' (version '1.0.3-preview5.19217.3') was successfully installed.